README.md

```diff
@@ -383,6 +383,41 @@ r.Walk(func(route *mux.Route, router *mux.Router, ancestors []*mux.Route) error
 })
 ```
 
+### Middleware
+
+
+Since **vX.Y.Z**, mux supports the addition of middlewares to a router, which are executed if a
```

Comment:
Replace "router" with `[Router](https://godoc.org/github.com/gorilla/mux#Router)` so that we link to the docs.